src/ProviderPR/SFA.DAS.ProviderPR.Api/AppStart/AddServiceRegistrationsExtension.cs

```diff
@@ -1,22 +1,26 @@
 ﻿using RestEase.HttpClientFactory;
 using SFA.DAS.Api.Common.Infrastructure;
 using SFA.DAS.Api.Common.Interfaces;
+using SFA.DAS.ProviderPR.Application.Queries.GetEasUserByEmail;
 using SFA.DAS.ProviderPR.Application.Queries.GetRelationships;
 using SFA.DAS.ProviderPR.Infrastructure;
 using SFA.DAS.SharedOuterApi.Configuration;
 using SFA.DAS.SharedOuterApi.Infrastructure;
 using SFA.DAS.SharedOuterApi.Interfaces;
 using SFA.DAS.SharedOuterApi.Provider.DfeSignIn.Auth.Application.Queries.ProviderAccounts;
 using SFA.DAS.SharedOuterApi.Services;
+using System.Diagnostics.CodeAnalysis;
 
 namespace SFA.DAS.ProviderPR.Api.AppStart;
 
+[ExcludeFromCodeCoverage]
 public static class AddServiceRegistrationsExtension
 {
     public static IServiceCollection AddServiceRegistration(this IServiceCollection services, IConfigurationRoot configuration)
     {
         services.AddMediatR(c => c.RegisterServicesFromAssembly(typeof(GetRoatpV2ProviderQuery).Assembly));
         services.AddMediatR(c => c.RegisterServicesFromAssembly(typeof(GetRelationshipsQuery).Assembly));
+        services.AddMediatR(c => c.RegisterServicesFromAssembly(typeof(GetEasUserByEmailQuery).Assembly));
```

Comment:
This is not needed given the assembly is already being registered on the line above.

Reply:
ok removed